### PR TITLE
VEP annotation tweaks.

### PR DESCRIPTION
**Pull request by bpblanken**

### CODE REVIEW

File: v03_pipeline/lib/annotations/shared.py [hunk 1]
```diff
@@ -98,10 +72,12 @@
     'unprocessed_pseudogene',
     'artifact',
     'lincRNA',
+    'lincrna',
```
Comment (bpblanken) [inline v03_pipeline/lib/annotations/shared.py:75]: I... am not sure fi we expect other `RNA` case sensitivity issues.  I think we'll find out soon enough.

File: v03_pipeline/lib/annotations/shared.py [hunk 2]
```diff
@@ -8,33 +8,6 @@
...
-    'S',
-    'T',
-    'V',
-    'W',
-    'Y',
-    'X',
-    '*',
-    'U',
-]
-AMINO_ACIDS_LOOKUP = hl.dict(hl.enumerate(AMINO_ACIDS, index_first=False))
```
Comment (bpblanken) [inline v03_pipeline/lib/annotations/shared.py:36]: There's weird stuff buried in the amino acid string.

For example:

```
G/GPAPPPSLHSLPSX
```

or 

```
L/LGX
```

I think this is supposed to mean `L` -> `L` OR `G` or `X`?

Comment (bpblanken) [reply v03_pipeline/lib/annotations/shared.py:36]: I just ripped out the enumification so we could get a run in and see what's there.

Comment (hanars) [reply v03_pipeline/lib/annotations/shared.py:36]: can we run off this branch or do we need to merge this to master first?

Comment (bpblanken) [reply v03_pipeline/lib/annotations/shared.py:36]: Yeah, I (already did) can run off this branch.  It's a PR into a branch anyways... no rush here.

**Pull request closed without merging**